src/CMakeLists.txt

```diff
@@ -50,6 +50,10 @@ function(valhalla_module)
   create_source_groups("Header Files" ${MODULE_HEADERS})
   create_source_groups("Source Files" ${MODULE_SOURCES})
 
+  target_compile_definitions(${library} PRIVATE
```

Comment:
why duplicate this, cant you just add the two lines below it into the existing compile definitions below?